### ISSUE TITLE
A separate git repo for configuration files

### ISSUE THREAD

**Issue by Srheft** [desihub/desispec#656]
The parameters and reference values in the configuration files could change multiple times within
one desispec release. 
Versioning the config files has been a long standing discussion but an optimum way of doing so was never finalized. Ql software maintains its separate monthly version but that does not include versioning configuration files. It seems that having a separate *config* repo would help tracking all the config versions independent from the code. This may impose a series of potential problems. Is there a more popular/practical alternative?

**Comment by julienguy** [comment]
Yes, a separate git repo is the best choice if this repo will not contain any large or binary files (like fits).
Otherwise a SVN repo would be better, like the one we have for the calibration data.

**Comment by rkehoe** [comment]
@julienguy, @srheft  This ticket seems to be still open.   In answer to your point, the configuration files will not be binary files and should in general be relatively short files describing the QL pipelines and the input parameters needed to drive them.  I do not see a separate repo in desi hub has been created for this purpose, but I agree it would be a good idea to assist with version.

I'm tagging each of you here for a response this week.  I can imagine that there are other reasons to have a 'config' repo, for other desi hub codes, but if there are no objections, I will create a 'qlconfig' repo at the end of the week.  @sbailey, you might also want to comment on this before we proceed.

**Comment by rkehoe** [comment]
@sbailey  Will you create that?  I'm happy to do it, but I don't appear to have permission to create a repo in desihub.

**Comment by sbailey** [comment]
@rkehoe I just created the qlconfig repo and granted write permission to the DESI developer team.
